net-ftp/atftp/atftp-0.8.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools flag-o-matic systemd
+
+DESCRIPTION="Advanced TFTP implementation client/server"
+HOMEPAGE="https://sourceforge.net/projects/atftp/"
+SRC_URI="mirror://sourceforge/atftp/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE="selinux tcpd readline pcre"
+
+DEPEND="tcpd? ( sys-apps/tcp-wrappers )
+	readline? ( sys-libs/readline:0= )
+	pcre? ( dev-libs/libpcre2 )"
+RDEPEND="${DEPEND}
+	!net-ftp/tftp-hpa
+	!net-ftp/uftpd
+	selinux? ( sec-policy/selinux-tftp )"
+BDEPEND="${DEPEND}"
```

Comment:
Are you sure that all of DEPEND are also BDEPEND?

Reply:
@Flowdalic IMHO all three of 

```
DEPEND="tcpd? ( sys-apps/tcp-wrappers )
    readline? ( sys-libs/readline:0= )
    pcre? ( dev-libs/libpcre2 )"
```

are needed at build time, yes.
Do you have a different opinion?

Reply:
Yes they are build dependencies, but I suspect they are not BDEPEND dependencies, just DEPEND dependencies.